include/tup-utils/stopTimeStore.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+#include <cista/containers/mmap_vec.h>
+#include <cista/containers/paged.h>
+#include <cista/containers/paged_vecvec.h>
+#include <cista/mmap.h>
+#include <cista/strong.h>
+
+#include <filesystem>
+#include <optional>
+#include <string>
+
+using stop_time_idx_t = cista::strong<std::uint32_t, struct stoptime_idx>;
+
+/**
+ * An Event when a vehicle of a trip arrives or ends at a stop
+ */
+struct stopTime {
```

Comment:
do we need a date here, to have a unique date, trip and stop combination?

Reply:
Yes indeed. I added the attribute, thanks for the heads up!